ubi-ruby-builder/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Ruby Builder
+FROM registry.access.redhat.com/ubi8/ubi
+ARG RUBY_MAJOR_VERSION
+ARG RUBY_FULL_VERSION
+ARG RUBY_SHA256=6c0bdf07876c69811a9e7dc237c43d40b1cb6369f68e0e17953d7279b524ad9a
+
+RUN yum install -y --setopt=tsflags=nodocs gcc \
+                                           gcc-c++ \
+                                           make \
+                                           openssl-devel \
+                                           http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/readline-devel-7.0-10.el8.x86_64.rpm \
```

Comment:
This line should be on a separate `RUN` since you're downloading something very differently from the rest of the packages.